Microsoft.AVS.VMFS/Microsoft.AVS.VMFS.psm1

```diff
@@ -69,12 +69,12 @@ function Set-VmfsIscsi {
         throw "Invalid SCSI IP address $ScsiIpAddress provided."
     }
 
-    $OperationProtectionSource = Get-Content -Path ".\OperationProtection.cs"
+    $OperationProtectionSource = Get-Content -Path "..\Microsoft.AVS.Management\OperationProtection.cs"
```

Comment:
This is not going to work when we install the module, it needs to be embedded. Something like [this](https://blog.stefan-gossner.com/2010/05/07/using-csharp-c-code-in-powershell-scripts/)?

Reply:
Sounds good, I've added it as embedded in the VMFS psm1 file. I was initially thinking of adding the embedded code in a separate OperationProtection.ps1 file under the Microsoft.AVS.Management folder and source it in VMFS module similar to https://github.com/Azure/Microsoft.AVS.Management/blob/373858f98d61caa668ef8212b48a511f7e6ab6a5/Microsoft.AVS.Management/Microsoft.AVS.Management.psm1#LL2C30-L2C30, but I wasn't sure if the VMFS module file would have access to the other folders when installing the module?

Reply:
Sounds good, I've added the embedded code in the Microsoft.AVS.Management module instead